apps/ledger-live-desktop/tests/page/modal/add.account.modal.ts

```diff
@@ -25,12 +25,37 @@ export class AddAccountModal extends Modal {
   async selectToken(token: Token) {
     await this.selectAccount.click();
     await this.selectAccountInput.fill(token.tokenName);
-    await this.selectTokenNetwork(token).click();
+    if (await this.selectTokenNetwork(token).isVisible()) {
+      await this.selectTokenNetwork(token).click();
+    } else {
+      const dropdownSize: any = await this.dropdownOptionsList.boundingBox();
```

Comment:
any > [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise)<[null](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null) | [Object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object)>

Reply:
It doesn't work :(